google-cloud-firestore/src/main/java/com/google/cloud/firestore/Transaction.java

```diff
@@ -51,6 +51,16 @@
     T updateCallback(Transaction transaction) throws Exception;
   }
 
+  /**
+   * User callback that takes a Firestore Async Transaction
```

Comment:
"Async user callback that takes a Firestore transaction."

Can you also add a period to line 45?

Reply:
period was added to both